features/step_definitions/metering.rb

```diff
@@ -50,7 +50,7 @@
   step %Q/all metering related pods are running in the "#{cb.metering_namespace.name}" project/
   step %Q/I wait for the "#{metering_name}" metering to appear/
   # added check for datasource to make sure promethues imports are working
-  step %Q/all datasources for "<%= cb.metering_namespace.name %>" project are imported/
+  step %Q/all reportdatasources are importing from Prometheus/
```

Comment:
Didn't we agree on a neutral name here? Also why one word `reportdatasources`?

Reply:
anyway, merging, will let you make nicer if you wish

Reply:
I found out that datasources is actually a shorthand for reportdatasources.  I decided to use the full name to be more explicit.